src/cmd/servo.rs

```diff
@@ -138,7 +142,16 @@ pub fn run_list(args: &ArgsList) -> Result<()> {
         println!("{}", keys.join(" "));
         return Ok(());
     }
-    println!("{}", list);
+    if args.json {
+        println!("{}", list);
+        return Ok(());
+    }
+    println!("usb_sysfs_path\tserial\tproduct");
+    let mut devices = list.devices().clone();
+    devices.sort_by(|l, r| l.usb_sysfs_path().cmp(r.usb_sysfs_path()));
+    for s in devices {
+        println!("{}\t{}\t{}", s.usb_sysfs_path(), s.serial(), s.product());
```

Comment:
I probably want to it in the following order: product, serial, sysfs_path.

But this is a highly subjective matter, I guess. If the current order makes sense for you, that's fine.

By the way, it would be nice to have the column aligned, meaning determining a reasonable width for each field instead of just using `\t`. Right now, it looks like this for me:
```
usb_sysfs_path	serial	product
/sys/bus/usb/devices/1-6.1	G1911051939	Servo V4
/sys/bus/usb/devices/1-6.2	0780605B-8D25B743	Cr50
```

Reply:
Thanks for the suggestion! Now it looks like this as suggested:
```
(workstation) $ lium servo list
product         serial                          usb_sysfs_path
Servo V4p1      SERVOV4P1-S-2202180988          /sys/bus/usb/devices/1-4.5
```